### ISSUE TITLE
SBT file

### ISSUE THREAD

**Issue by omidb** [travisbrown/syzygist#4]
Hi,

In the SBT file both module names are the same and are "syzygist-split":
```scala
lazy val split = project.settings(moduleName := "syzygist-split")
  .settings(commonSettings: _*)
  .settings(scalacOptions += "-Xfatal-warnings")

lazy val parse = project.settings(moduleName := "syzygist-split")
  .settings(commonSettings: _*)
  .settings(libraryDependencies += "org.parboiled" %% "parboiled" % "2.1.0")
```

I have problem resolving the dependency of Split package (for running the example in readme).
Is there anything specific that I have to do for using it in IntelliJ?

Thanks!

**Comment by travisbrown** [comment]
Ah, good catch!—that's a typo. If you want to submit a pull request I'll happily merge it, or I can do it later today. Thanks.

**Comment by travisbrown** [comment]
Fixed by #5.
